ArcGISRuntimeSDKQt_CppSamples/Scenes/Animate3DSymbols/Animate3DSymbols.h

```diff
@@ -34,37 +34,35 @@ namespace Esri
   }
 }
 
-class QAbstractItemModel;
-
-#include "Point.h"
+class QAbstractListModel;
+class MissionData;
 
 #include <QQuickItem>
 #include <QString>
 
-#include <cmath>
 #include <memory>
 
 class Animate3DSymbols : public QQuickItem
 {
-  Q_OBJECT
-
   Q_PROPERTY(bool missionReady READ missionReady NOTIFY missionReadyChanged)
   Q_PROPERTY(int missionSize READ missionSize NOTIFY missionSizeChanged)
-  Q_PROPERTY(int missionFrame READ missionFrame WRITE setMissionFrame)
+  Q_PROPERTY(int missionFrame READ missionFrame WRITE setMissionFrame NOTIFY missionFrameChanged)
+  Q_PROPERTY(QAbstractListModel* missionModel READ missionsModel CONSTANT)
```

Comment:
typo here I think - the READ should be `missionModel` like the property name?

```suggestion
  Q_PROPERTY(QAbstractListModel* missionModel READ missionModel CONSTANT)
```